members/CusB0abdc0e61526058673.yaml

```diff
@@ -0,0 +1,130 @@
+bioguide: CusB0abdc0e61526058673
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: "https://www.regulations.gov/comment?D=ATF-2018-0002-0001"
+    - wait:
+      - value: 6
+    - find:
+      - selector: div.GIY1LSJGSC form
+    - fill_in:
+        - name: message
+          selector: textarea#x-auto-0-input
+          value: $MESSAGE
+          required: true
+        - name: first
+          selector: input#x-auto-1-input
+          value: $NAME_FIRST
+          required: true
+        - name: last
+          selector: input#x-auto-2-input
+          value: $NAME_LAST
+          required: true
+        - name: street
+          selector: input#x-auto-3-input
+          value: $ADDRESS_STREET
+          required: true
+        - name: street2
+          selector: input#x-auto-4-input
+          value: $ADDRESS_STREET_2
+          required: false
+        - name: city
+          selector: input#x-auto-5-input
+          value: $ADDRESS_CITY
+          required: true
+        - name: zip5
+          selector: input#x-auto-8-input
+          value: $ADDRESS_ZIP5
+          required: true
+        - name: email
+          selector: input#x-auto-10-input
+          value: $EMAIL
+          required: false
+        - name: phone
+          selector: input#x-auto-11-input
+          value: $PHONE
+          required: false
+    - select:
+        - name: state
+          selector: input#x-auto-6-input
```

Comment:
this element is an `input` and not a `select` - does this correctly enter the values? in the past we've sometimes had to handle these differently if i remember correctly